code/game/turfs/open/floor/plating/wasteplanet.dm

```diff
@@ -32,31 +32,56 @@
 	desc = "Corrupted steel."
 	icon_state = "plating_rust"
 
-/turf/open/floor/wood/waste
-	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
-	planetary_atmos = TRUE
-
 /turf/open/indestructible/hierophant/waste
 	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
 	planetary_atmos = TRUE
+	light_color = LIGHT_COLOR_FLARE
 
 /turf/open/indestructible/hierophant/two/waste
 	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
 	planetary_atmos = TRUE
 	light_color = LIGHT_COLOR_FLARE
 
-/turf/open/water/waste
+/turf/open/floor/wood/waste
 	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
 	planetary_atmos = TRUE
 
-/turf/open/floor/plating/grass/wasteplanet
-	icon_state = "junglegrass"
+///Cement Turfs
+
+/turf/open/floor/concrete/wasteplanet
 	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
 	planetary_atmos = TRUE
+	baseturfs = /turf/open/floor/plating/asteroid/wasteplanet
 
-/turf/open/floor/plating/dirt/old/waste
+/turf/open/floor/concrete/wasteplanet/Initialize()
+	. = ..()
+	icon_state = pick(list(
+		"conc_smooth",
+		"conc_slab_1",
+		"conc_slab_2",
+		"conc_slab_3",
+		"conc_slab_4",
+		"conc_tiles"
+	))
+
+/turf/open/floor/concrete/wasteplanet/MakeSlippery(wet_setting, min_wet_time, wet_time_to_add, max_wet_time, permanent)
```

Comment:
why is specifically wasteplanet concreate not slippery

Reply:
wasteplanets can spawn with water vapor in the atmosphere (unless this has been changed. I did not check), and having a biome turn into a giant slip and slide doesn't sound like an engaging gameplay experience.